src/composer.cpp

```diff
@@ -246,6 +247,11 @@ void CcomposerBackend::SetNotePercussive(int const voice, int const note)
 //---------------------------------------------------------
 void CcomposerBackend::SetNoteMelodic(int const voice, int const note)
 {
+    if (voice >= 9)
```

Comment:
Is it possible to use `kNumMelodicVoices` class constant? (and in other places)
```suggestion
    if (voice >= kNumMelodicVoices)
```

Reply:
Sure, looks more pretty